lib/saml_idp/xml_security.rb

```diff
@@ -136,55 +139,82 @@ def validate_doc_params_signature(base64_cert, soft = true, params)
           sig_alg: params[:SigAlg]
         )
         log '***** validate_doc_params_signature: verify_signature:'
-        verify_signature(base64_cert, params[:SigAlg], Base64.decode64(params[:Signature]), canon_string, soft)
+        verify_signature(
+          base64_cert,
+          params[:SigAlg],
+          Base64.decode64(params[:Signature]),
+          canon_string, soft
+        )
```

Comment:
seems weird that it split most onto different lines but not all? I use an "all same or all different" strat
```suggestion
        verify_signature(
          base64_cert,
          params[:SigAlg],
          Base64.decode64(params[:Signature]),
          canon_string,
          soft,
        )
```

Reply:
oh, this one i think i fixed manually and just missed those last two (i thought the formatting it suggested was hard to read), thanks for catching!

Reply:
fixed in https://github.com/18F/saml_idp/pull/98/commits/2b8c1afa357e64a05ddf8932dfae209da2a40a6a